internal/k8s/controller.go

```diff
@@ -2991,6 +2991,13 @@ func getEndpointsBySubselectedPods(targetPort int32, pods []*api_v1.Pod, svcEps
 	return endps
 }
 
+func ipv6SafeAddrPort(addr string, port int32) string {
```

Comment:
same here. Woudn't using `net.JoinHostPort` be cleaner ?

Reply:
ditto. Thanks